### ISSUE TITLE
Crash when playing on a thin world

### ISSUE THREAD

**Issue by WizardHorse** [anegostudios/VintageStory-Issues#601]
Moved Polar Equator distance to mildly short distance. (however this doesnt seem to matter)
(World Size : 10240x256x32)

When reloading world it claims that it is out of date and needs fixing. This does not stop the crashing.

I also tried this on a 10240x256x512 world size.
When I got to the world border and ran away it crashed.

I also tried this on a 32x256x32 world size.
I got the exact same fatal error when falling off the world where my player was below the world ( I could see the bottom blocks of the world)
**Modded:** No

```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.13.4 (Stable)30-09-20 5:36:57 PM: Critical error occurred
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: dx/dz must be within [0,2]
   at Vintagestory.GameContent.MultiChunkMapComponent.unsetChunk(Int32 dx, Int32 dz)
   at Vintagestory.GameContent.ChunkMapLayer.OnViewChangedClient(List`1 nowVisible, List`1 nowHidden)
   at Vintagestory.GameContent.WorldMapManager.onViewChangedClient(List`1 nowVisible, List`1 nowHidden)
   at Vintagestory.GameContent.GuiElementMap.EnsureMapFullyLoaded()
   at Vintagestory.GameContent.GuiDialogWorldMap.<ComposeDialog>b__16_0(Single dt)
   at Vintagestory.Common.EventManager.TriggerGameTick(Int64 ellapsedMilliseconds, IWorldAccessor world)
   at Vintagestory.Client.NoObf.ClientMain.MainRenderLoop(Single dt)
   at Vintagestory.Client.NoObf.ClientMain.MainGameLoop(Single deltaTime)
   at _VtIDI6oYFpyEcQSZBFso9UVxS9aA._ZnYzeuHqveMMiEvOnpahncAMjfg(Single )
   at _LSAAgZhw31Rqs2is6nnojfZBbQC._4BykKOCicJqCBHpyWIgTmxWrTrg(Single )
   at _LSAAgZhw31Rqs2is6nnojfZBbQC._6XmOxKoTYrFLRCGd58p58XW43nD(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _lYLQLGyldvbDG9rPQyiRVeIqQVp._EEzpApWBQHtHR4iMJabXK2wDvlA(_ex0VSgRAdtsvhANmJHZHaf9ATWt , String[] )
   at _IcGByoJhapvESnm6vaOawAHbkeU._EEzpApWBQHtHR4iMJabXK2wDvlA(ThreadStart )
-------------------------------

Event Log, last 3 entries
==================================
{ TimeGenerated = 30-09-20 5:36:57 PM, Site = , Source = .NET Runtime, Message = Application: Vintagestory.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.ArgumentOutOfRangeException
   at Vintagestory.GameContent.MultiChunkMapComponent.unsetChunk(Int32, Int32)
   at Vintagestory.GameContent.ChunkMapLayer.OnViewChangedClient(System.Collections.Generic.List`1<Vintagestory.API.MathTools.Vec2i>, System.Collections.Generic.List`1<Vintagestory.API.MathTools.Vec2i>)
   at Vintagestory.GameContent.WorldMapManager.onViewChangedClient(System.Collections.Generic.List`1<Vintagestory.API.MathTools.Vec2i>, System.Collections.Generic.List`1<Vintagestory.API.MathTools.Vec2i>)
   at Vintagestory.GameContent.GuiElementMap.EnsureMapFullyLoaded()
   at Vintagestory.GameContent.GuiDialogWorldMap.<ComposeDialog>b__16_0(Single)
   at Vintagestory.Common.EventManager.TriggerGameTick(Int64, Vintagestory.API.Common.IWorldAccessor)
   at Vintagestory.Client.NoObf.ClientMain.MainRenderLoop(Single)
   at Vintagestory.Client.NoObf.ClientMain.MainGameLoop(Single)
   at _VtIDI6oYFpyEcQSZBFso9UVxS9aA._ZnYzeuHqveMMiEvOnpahncAMjfg(Single)
   at _LSAAgZhw31Rqs2is6nnojfZBbQC._4BykKOCicJqCBHpyWIgTmxWrTrg(Single)
   at _LSAAgZhw31Rqs2is6nnojfZBbQC._6XmOxKoTYrFLRCGd58p58XW43nD(Single)
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(System.Object, OpenTK.FrameEventArgs)
   at System.EventHandler`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Invoke(System.Object, System.__Canon)
   at OpenTK.GameWindow.RaiseRenderFrame(Double, Double ByRef)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double, Double)
   at _lYLQLGyldvbDG9rPQyiRVeIqQVp._EEzpApWBQHtHR4iMJabXK2wDvlA(_ex0VSgRAdtsvhANmJHZHaf9ATWt, System.String[])
   at _IcGByoJhapvESnm6vaOawAHbkeU._EEzpApWBQHtHR4iMJabXK2wDvlA(System.Threading.ThreadStart)

 }
```

**Comment by WizardHorse** [comment]
I dont know how to use github so yee its a bug i would say. I only made it to submit this bug.

**Comment by tyronx** [comment]
thanks for the report, this should be fixed in v1.14
